calibr8/contrib/studentt.py

```diff
@@ -340,11 +350,19 @@ def predict_dependent(self, x, *, theta=None):
         """
         if theta is None:
             theta = self.theta_fitted
-        mu = core.exponential(x, theta[:2])
+
+        if self.fixed_intercept is None:
+            theta_mu = (theta[0], theta[1], theta[2])
+            theta_scale = theta[3:]
```

Comment:
this is not correct! It must be theta[3:-1]. Same further down! theta[-1] is the df

Reply:
Also, the line above could be theta[:3] here. Sorry, the suggestion mode did not work, so I just commented the mistakes.